src/vm.rs

```diff
@@ -234,8 +234,7 @@ impl<T: GuestStagePageTable> Vm<T, VmStateInitializing> {
                 // Fake compound device identifier (DICE CDI)
                 // TODO Get the CDI from e.g. the TSM driver.
                 b"THISISARANDOMCDI",
-                // TODO Use a TVM unique address, e.g. the TVM state one.
-                0,
+                state_addr,
```

Comment:
I'm not sure the guest physical address of the start of RAM really qualifies as unique since, by convention, RAM usually starts at 0x8000_0000 in the (guest) physical address space.

Does this identifier really need to be an address? Could you use, say, the PageOwnerId of the VM instead?

Reply:
Yes, we can use the `PageOwnerId`. As long as it's unique per VM, it's good. Thanks for the suggestion.